person.rb

```diff
@@ -1,8 +1,8 @@
-require_relative './corrector'
-require_relative './rental'
+require_relative 'corrector'
+require_relative 'rental'
 
 class Person
-  attr_accessor :name, :age, :rental
+  attr_accessor :name, :age, :rentals, :parent_permission
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] One of your parameters in line 8 is wrong, I suggest to take a careful look to it and fix it.